docs/pages/bssw/bssw_content_publishing.md

```diff
@@ -42,10 +42,13 @@ The following steps assume that the BSSw editorial team has already determined t
 * Hyperlinks validity.
 * Check for any prior versions of the same content. If the prior version is very outdated, consider replacing it with this newer content. Please judge on a case-by-case basis.
 * Past events of the same conferences/workshops/etc. are not deleted and just kept as archives.
+* If using [formal citations/references](bssw_styling_originalarticles.html#citationsreferences), use [`wikize_refs.py -i <base>.md`](https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy) to deal with these (see [here](https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/ReferencesInMarkdownHybridApproach.md) for details).
 
 ### Pre-publishing Checks
 * Finalize the category(ies), topic(s) and other parameters in metadata.
-* Check on preview site to confirm overall display ("publish: preview" in metadata).
-* Change "publish: yes" in metadata before merging into master.
+* Ensure to run [`wikize_refs.py -i <base>.md`](https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/ReferencesInMarkdownHybridApproach.md) and commit before previewing (if `wikize_refs.py` is being used).
```

Comment:
Ensure to run --> Be sure to run

or

Ensure to run [`wikize_refs.py -i <base>.md`] --> Ensure [`wikize_refs.py -i <base>,md`] is run

Reply:
Thanks!  Addressed in commit https://github.com/betterscientificsoftware/bssw.io/pull/750/commits/a1eab3be92286d232789f159d147022d1b825950.